components/InputField.js

```diff
@@ -181,10 +170,12 @@ class InputField extends React.Component {
   }
 
   validate(value) {
-    if (!value) {
+    const type = this.props.type || 'text';
+
+    if (!value && type !== 'datetime-local') {
```

Comment:
Why ignore the `this.props.required` flag for `datetime-local`?

Reply:
Ops, sorry I was trying something and committed this change by mistake. This should not have been changed. I've reverted this. 🤔 🤯